docs/marketplace/dev/feature-integration-guides/202108.0/combined-product-offer-import-integration.md

```diff
@@ -0,0 +1,2185 @@
+     *
+     * @return \Pyz\Zed\DataImport\Business\Model\DataImporterConditional
+     */
+    public function getConditionalCsvDataImporterFromConfig(
+        DataImporterConfigurationTransfer $dataImporterConfigurationTransfer
+    ): DataImporterConditional {
+        $csvReader = $this->createCsvReaderFromConfig($dataImporterConfigurationTransfer->getReaderConfiguration());
+
+        return $this->createDataImporterConditional($dataImporterConfigurationTransfer->getImportType(), $csvReader);
+    }
+
+    /**
+     * @param string $importType
+     * @param \Spryker\Zed\DataImport\Business\Model\DataReader\DataReaderInterface $reader
+     *
+     * @return \Pyz\Zed\DataImport\Business\Model\DataImporterConditional
+     */
+    public function createDataImporterConditional(
+        string $importType,
+        DataReaderInterface $reader
+    ): DataImporterConditional {
+        return new DataImporterConditional($importType, $reader, $this->getGracefulRunnerFacade());
+    }
+
+    /**
+     * @return \Spryker\Zed\DataImport\Business\Model\DataImportStep\DataImportStepInterface
+     */
+    public function createCombinedProductOfferReferenceToIdProductOfferStep(): DataImportStepInterface
+    {
+        return new CombinedProductOfferReferenceToIdProductOfferStep(
+            $this->getProductOfferFacade()
+        );
+    }
+
+    /**
+     * @return \Spryker\Zed\DataImport\Business\Model\DataImportStep\DataImportStepInterface
+     */
+    public function createCombinedStockNameToIdStockStep(): DataImportStepInterface
+    {
+        return new CombinedStockNameToIdStockStep();
+    }
+
+    /**
+     * @return \Spryker\Zed\DataImport\Business\Model\DataImportStep\DataImportStepInterface
+     */
+    public function createCombinedProductOfferStockWriterStep(): DataImportStepInterface
+    {
+        return new CombinedProductOfferStockWriterStep();
+    }
+
+    /**
+     * @return \Pyz\Zed\DataImport\Business\Model\DataSet\DataSetConditionInterface
+     */
+    public function createCombinedProductOfferStockMandatoryColumnCondition(): DataSetConditionInterface
+    {
+        return new CombinedProductOfferStockMandatoryColumnCondition();
+    }
+}
+
+```
+
+</details>
+
+
+<details>
+<summary markdown='span'>src/Pyz/Zed/ProductOfferStockDataImport/Business/ProductOfferStockDataImportFacade.php</summary>
+
+```php
+
+<?php
+
+namespace Pyz\Zed\ProductOfferStockDataImport\Business;
+
+use Generated\Shared\Transfer\DataImporterConfigurationTransfer;
+use Generated\Shared\Transfer\DataImporterReportTransfer;
+use Spryker\Zed\ProductOfferStockDataImport\Business\ProductOfferStockDataImportFacade as SprykerProductOfferStockDataImportFacade;
+
+/**
+ * @method \Pyz\Zed\ProductOfferStockDataImport\Business\ProductOfferStockDataImportBusinessFactory getFactory()
+ */
+class ProductOfferStockDataImportFacade extends SprykerProductOfferStockDataImportFacade implements ProductOfferStockDataImportFacadeInterface
```

Comment:
ProductOfferStockDataImportFacadeInterface is not presented in this guide

Reply:
added